src/Language/PureScript/Parser/Lexer.hs

```diff
@@ -440,12 +444,6 @@ mname = token go P.<?> "module name"
   go (UName s) | validModuleName s = Just s
   go _ = Nothing
 
-uname' :: String -> TokenParser ()
-uname' s = token go P.<?> show s
-  where
-  go (UName s') | s == s' = Just ()
-  go _ = Nothing
```

Comment:
Deleted this as it was unused, and I wasn't sure what the right implementation would be anymore: should the `validUName` predicate apply? etc.